test/CMakeLists.txt

```diff
@@ -69,10 +69,10 @@ set_target_properties(build_timezones PROPERTIES FOLDER "Tests")
 
 add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/test/data/utrecht_tiles/0/003/196.gph
   COMMAND ${CMAKE_COMMAND} -E make_directory test/data/utrecht_tiles/
-  COMMAND $<TARGET_FILE:valhalla_build_tiles>
+  COMMAND ${CMAKE_BINARY_DIR}/valhalla_build_tiles
```

Comment:
I made this change because e.g. Xcode uses per-buildtype output directories and I couldn't build various targets because the path above is incorrect. However, I agree that this change is unrelated to GraphTiles, so we can move it to a different patch. It seems like the `valhalla_build_tiles` target doesn't exist on Windows at all.